inference-engine/tests_deprecated/unit/engines/gna/gna_matcher.cpp

```diff
@@ -436,7 +436,7 @@ void GNAPluginAOTMatcher :: match() {
     // matching gna_propagate forward call.
     MockICNNNetwork net;
     
-    size_t weightsSize = 440*3;
+    size_t weightsSize = 656384;
```

Comment:
Can you explain this change please?

Reply:
`GNAPluginAOTMatcher` allocate same empty blob for all tests. So I changes it to allocate blob just enough size to "store" weights for In `maxpoolAfterRelu` model.